tests/strategies/test_strategy_pv.py

```diff
@@ -322,11 +334,13 @@ def area_test66():
 
 @pytest.fixture()
 def pv_test66(area_test66):
+    GlobalConfig.FUTURE_MARKET_DURATION_HOURS = 0
```

Comment:
Is this for making the tests run faster or in order to not have to populate the `future_markets` member in the area mock/fixture?

Reply:
The reason why I did that specifically for this test was because this test was actually reading profiles outside the range of one day. If you check the test implementation, you will see that the goal of this is to check that the energy production of the PV during the night is lower than the production during the night. Unfortunately this is done with this check:
`elif time > afternoon_time:`
which takes into account timestamps from the next day (because taking into account the future markets the profile is 2 days long). Quick fix was to completely disregard the future markets in this test (because this is not the goal of the test at the very end)